cypress/fixtures/test-app/index.html

```diff
@@ -89,6 +92,24 @@ <h2>*AllByText</h2>
     <h2>*ByText on another page</h2>
     <a onclick='setTimeout(function() { window.location = "/cypress/fixtures/test-app/next-page.html"; }, 100);'>Next Page</a>
   </section>
+  <section>
+    <h2>Eventual existence</h2>
+    <button id="eventually-will-exist"></button>
+    <script>
+      setTimeout(() => {
+        document.querySelector('#eventually-will-exist').innerHTML = 'Eventually Exists'
+      }, 500)
+    </script>
```

Comment:
This is brilliant

Reply:
Thanks, it seems a bit hacky, but the [colocation](https://kentcdodds.com/blog/colocation) more easily shows intent.